app/models/induction_record.rb

```diff
@@ -47,8 +47,9 @@ class InductionRecord < ApplicationRecord
     withdrawn: "withdrawn",
   }, _prefix: "training_status"
 
-  scope :fip, -> { joins(:induction_programme).merge(InductionProgramme.full_induction_programme) }
   scope :cip, -> { joins(:induction_programme).merge(InductionProgramme.core_induction_programme) }
+  scope :fip, -> { joins(:induction_programme).merge(InductionProgramme.full_induction_programme) }
+  scope :sffip, -> { joins(:induction_programme).merge(InductionProgramme.school_funded_fip) }
```

Comment:
```suggestion
  scope :school_funded_fip, -> { joins(:induction_programme).merge(InductionProgramme.school_funded_fip) }
```
Think this would help make it easier to understand

Reply:
Sure